src/jobs/index.ts

```diff
@@ -85,25 +86,27 @@ export async function handleGithubJobs(
   tx.finish();
 }
 
-export const opts = {
-  schema: {
-    body: {
-      type: 'object',
-      required: ['message'],
-      properties: {
-        message: {
-          type: 'object',
-          required: ['data'],
-          properties: {
-            data: {
-              type: 'string',
-            },
-          },
-        },
-      },
-    },
-  },
-};
+// Error handling wrapper function for Cron Jobs
+// Additionally handles Auth from Cloud Scheduler
+export async function handleCronJobs(
+  handler,
+  request: FastifyRequest,
+  reply: FastifyReply
+) {
+  const verified = await handleCronAuth(request, reply);
```

Comment:
Does this framework not provide a native way to authenticate requests rather than hoping every developer remembers to add authentication in their application code in every endpoint ?

Reply:
The auth for jobs are handled via cloud scheduler (managed by terraform), so developers don't need to worry about it

Reply:
The webhooks do currently require auth for each endpoint, but I'm not sure if there is a way around that from a security perspective, since traffic is open to the internet

Reply:
What I meant was whether the web framework we use supports that rather than implementing auth inside the application logic.

Reply:
Fastify, the web framework eng-pipes uses does not (as far as I am aware) natively support hmac auth. With that said, I do think we could potentially do a better job enforcing that webhooks use hmac and reduce the amount of duplication across routes.

Reply:
This PR only touches routes for cron jobs which are all protected so this discussion isn't relevant here, but the main challenge for the normal webhook routes is that they currently each use their own secrets and header names. We shouldn't try to unify the secrets since that's a security vulnerability, but we could try to unify the headers & auth handlers into one function to reduce duplication